cookbook/security/voters_data_permission.rst

```diff
@@ -213,9 +212,7 @@ from the authorization checker is called.
             $post = ...;
 
             // keep in mind, this will call all registered security voters
-            if (false === $this->get('security.authorization_checker')->isGranted('view', $post)) {
-                throw new AccessDeniedException('Unauthorised access!');
-            }
+            $this->denyAccessUnlessGranted('view', $post, 'Unauthorised access!');
```

Comment:
Minor comment: I know that the original doc said `Unauthorised`, but we use the American English dialect in the Symfony docs, so this should be changed by `Unauthorized`. Thanks!